### ISSUE TITLE
Issue with lr-scummvm hardware acceleration on rpi4

### ISSUE THREAD

**Issue by dszakallas** [RetroPie/RetroPie-Setup#3925]
ScummVM recently added OpenGL(ES) acceleration and support is auto-detected at compile time. However I suppose there's an issue with the detection itself as it requests OpenGL context on rpi4, which is not available.

```
[ERROR] Requesting OpenGL context, but RetroArch is compiled against OpenGLES. Cannot use HW context.
[ERROR] [Environ]: SET_HW_RENDER - Dynamic request HW context failed.
[libretro WARN] Failed to set up hardware rendering, falling back to software.
```

This behavior is not really problematic, but results in an annoying pop-up `Failed to set HW rendering` on game launch.

**Comment by cmitu** [comment]
Thanks for the heads up. FWIW, there's no 'compile time detection', support for one or the other GL flavor is specified with an option, I guess OpenGL is selected as default. We'll adjust the compilation flags and see if this warning disappears.

In the future, please use the forum for reports, as indicated in the issue template.

**Comment by cmitu** [comment]
The _Makefile_ pre-sets the `HAVE_OPENGL[ES2]` flags depending on the `platform` parameter, ignoring any `HAVE_OPENGL/OPENGLES2` parameters passed to the `make` command. Using:
```
make platform=rpi4_64
```
will force OpenGLES2. I don't see how - besides patching the _Makefile_ - a hardware renderer choice can be passed on. We'll have to probably talk to upstream about this.

**Comment by cmitu** [comment]
@dszakallas this should be fixed with the recent changes in #3927. Can you update and give it a try ?

**Comment by dszakallas** [comment]
Seems to work for me. No popup and HW acceleration is now on in Video Settings. Thanks!
